srcpkgs/unzip/template

```diff
@@ -11,6 +11,10 @@ homepage="http://infozip.sf.net/"
 distfiles="${SOURCEFORGE_SITE}/infozip/${pkgname}60.tar.gz"
 checksum=036d96991646d0449ed0aa952e4fbe21b476ce994abc276e49d30e686708bd37
 
+alternatives="
+ unzip:unzip:/usr/bin/unzip
```

Comment:
this is wrong, the real binary must be renamed, otherwise you're trying to make an alternative from the same filename.

Reply:
Rename it to `infounzip`? 
I just removed the alternatives part for now.